cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java

```diff
@@ -110,25 +83,14 @@ public void clearAllData(){
     public void clearInternalData(){
         editor.remove(CONSENT_UUID_KEY);
         editor.remove(META_DATA_KEY);
-        editor.remove(EU_CONSENT__KEY);
+        editor.remove(EU_CONSENT_KEY);
         editor.remove(AUTH_ID_KEY);
         editor.commit();
     }
 
-
     public void clearConsentData(){
-        clearTCData();
-        editor.remove(IAB_CONSENT_CONSENT_STRING);
-    }
-
-    private void clearTCData() {
-        for(String s : dataSetFromStore()){
-            editor.remove(s);
-        }
-        editor.remove(TC_KEYS_KEY);
-    }
-
-    private String[] dataSetFromStore(){
-        return pref.getString(TC_KEYS_KEY, "").split(TC_KEYS_DELIMITER);
+        for(String key : pref.getAll().keySet()) if (key.startsWith(IABTCF_KEY_PREFIX))
```

Comment:
It's really just a matter of taste, but don't you think it looks better this way:
```Java
for(String key : pref.getAll().keySet()) 
  if (key.startsWith(IABTCF_KEY_PREFIX)) editor.remove(key);
```